docs/csharp/language-reference/keywords/types.md

```diff
@@ -16,19 +16,17 @@ The C# typing system contains the following categories:
 
 - [Pointer types](../../programming-guide/unsafe-code-pointers/pointer-types.md)
 
- Variables that are value types store data, and those that are reference types store references to the actual data. Reference types are also referred to as objects. Pointer types can be used only in [unsafe](unsafe.md) mode.
+ Variables that are value types store data, and those that are reference types store references to the actual data. Instances of reference types are also referred to as objects. Pointer types can be used only in [unsafe](unsafe.md) mode.
```

Comment:
This explanation, in general, isn't really working for me. These concepts are fundamental to both C# and .NET yet this description is very narrow and (IMO) confusing. I clicked on the description provided for reference types. It's a much better description. IMO, it's not a great description for reference types because it re-describes value and reference types again. It should focus on reference types.

I'd suggest we merge this PR and create an issue to address the feedback I've given. Thoughts?

Reply:
> I'd suggest we merge this PR and create an issue to address the feedback I've given.

Agree that the page under consideration can be improved further. Given its place in the TOC, it should rather describe and reference descendant articles.

The description of value/reference types seems to be more a concern of the "Concepts" part of the docs, not the "Language reference", though of course, they should link each other.

Reply:
@pkulikov Want me to run with my feedback or did you intend to do that?

Reply:
@richlander please create an issue with your feedback